src/app/image/image.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { I18n } from '../i18n/i18n';
+import { ImageService } from '../services/image/image.service';
+
+@Component({
+  selector: 'ru-img',
+  styleUrls: ['image.component.scss'],
+  templateUrl: 'image.component.html'
+})
+export class ImageComponent implements OnInit {
+  @Input() dimensions: string;
+  @Input() quality: number;
+  @Input() roundedCorners: number;
+  @Input() background: string;
+  @Input() watermark: string;
+  @Input() src: string;
+  displayedImage: string;
+  i18n = I18n;
+
+  constructor(private imageService: ImageService) {}
+
+  ngOnInit() {
```

Comment:
This seems complicated. Why not create a method which returns the right url instead?